README.md

```diff
@@ -49,13 +49,24 @@ Engineers typically work on a product team and may also belong to an _Engineerin
   - Slack: [#product-platform](https://artsy.slack.com/messages/product-platform) 🔒
```

Comment:
This channel can be updated to `#practice-platform` and the Practice lead can be updated to @ansor4 (above).

Reply:
Aha, thanks!